symphony/bdk/core/activity/help_command.py

```diff
@@ -0,0 +1,29 @@
+from symphony.bdk.core.activity.command import CommandContext, SlashCommandActivity
+from symphony.bdk.core.activity.registry import ActivityRegistry
+from symphony.bdk.core.service.message.message_service import MessageService
+from symphony.bdk.core.service.message.model import Message
+
+
+class HelpCommand(SlashCommandActivity):
+    """The help command is a particular CommandActivity which returns the list of all commands available for the
+    specific bot
+    The Slash command is triggered if we receive a MESSAGESENT event with message text:
+      - "@{bot_display name} {name}"
+    """
+
+    def __init__(self, registry: ActivityRegistry, callback: MessageService):
+        super().__init__("/help", True, callback, "List available commands")
+        self._registry = registry
+
+    def matches(self, context: CommandContext) -> bool:
```

Comment:
This is already implemented in the `SlashCommandActivity` class so this can be removed.